.github/workflows/integration.yml

```diff
@@ -13,6 +17,7 @@ jobs:
         secured: ["true"]
         entry:
           - { opensearch_version: 2.7.0 }
+          - { opensearch_version: 2.11.0 }
```

Comment:
when we do `pytest` in our development, do we check the OS version too? I just wanted to make sure we are not adding any blocker for the development. Like right now, we need OS 2.7 version during development. I assume with this PR, we won't have blocker like this. Could you please verify this?